app/poros/yelp_gym.rb

```diff
@@ -0,0 +1,10 @@
+class YelpGym
+  attr_reader :yelp_gym_id, :name, :address, :phone
+
+  def initialize(info)
+    @yelp_gym_id = info[:id]
+    @name = info[:name]
+    @address = info[:address]
+    @phone = info[:phone]
+  end
+end
```

Comment:
Because the BE is making the Yelp API call, we could maybe just call this a `Gym` poro and then create the `Gym` poros for the gym search by location and for the gym show page.

Reply:
[Issue](https://github.com/tvaroglu/spot_me_frontend/issues/20) opened